.travis.yml

```diff
@@ -1,9 +1,15 @@
 language: java
+# Sudo required for chrome as per https://docs.travis-ci.com/user/chrome
+sudo: required
```

Comment:
This should no longer be needed, there's a warning in Travis CI: "deprecated key sudo (The key `sudo` has no effect anymore.)".